src/index.js

```diff
@@ -24,12 +24,21 @@ const SyntaxHighlighter = (
     theme = opts.customTheme;
   }
 
+  if (opts.foldGutter) {
+    // `foldGutter` does not work with runmode and has to be done with the codeEditor component
+    // NOTE: To disable editing, to still allow folding, `readOnly` must be `true`.
```

Comment:
it's a little confusing, basically we have to force editmode to true for folding to work because it only works if we render a component—`runMode` doesn't support it. So it's a somewhat awkward API given turning on folding defacto enables editing... I left this comment here to remind people that are confused at that point that they can then also pass in readOnly if they still want folding but don't want editing.

Lmk you have a suggest to improve the comment readability